installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3122,7 +3122,7 @@ INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL)
 
 /* Plugin developed using PluginEngine V1 */
 INSERT INTO pac_config_properties (`cfkey`,`value`,`application`,`profile`,`label`,`createdBy`,`createdDate`,`modifiedBy`,`modifiedDate`)
- VALUES ('plugins.in.v1','gcp,redhat,contrast,checkmarx,crowdstrike','job-scheduler','prd','latest','System',now(),NULL,NULL) ON DUPLICATE KEY UPDATE value = 'gcp,redhat,contrast,checkmarx,crowdstrike', modifiedBy='System', modifiedDate=now();
+ VALUES ('plugins.in.v1','gcp,redhat,contrast,checkmarx,crowdstrike,wiz','job-scheduler','prd','latest','System',now(),NULL,NULL) ON DUPLICATE KEY UPDATE value = 'gcp,redhat,contrast,checkmarx,crowdstrike,wiz', modifiedBy='System', modifiedDate=now();
```

Comment:
Also should we move this as update to ee?

Reply:
You are right Santhoh. we need to move to EE.

Reply:
@arunpaladin @santhosh-challa @plyubich 
Shall I keep the value as GCP in the CE repository and move the entire query to the EE repository

Reply:
In OSS GCP is still following legacy. I think we may not need it. correct me @arunpaladin